### PR TITLE
More chain simulator tests

### PR DESCRIPTION
**Pull request by sstanculeanu**
## Reasoning behind the pull request
- more coverage needed
  
## Proposed changes
- bootstrapComponentsHolder tests 

## Testing procedure
- tests only

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: node/chainSimulator/components/bootstrapComponents_test.go [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+	}
+}
+
+func TestCreateBootstrapComponents(t *testing.T) {
+	t.Parallel()
+
+	t.Run("should work", func(t *testing.T) {
+		t.Parallel()
+
+		comp, err := CreateBootstrapComponents(createArgsBootstrapComponentsHolder())
```
Comment (iulianpascalau) [inline node/chainSimulator/components/bootstrapComponents_test.go:135]: should we call a comp.Close() at the end of the test?

File: node/chainSimulator/components/coreComponents_test.go [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+	}
+}
+
+func TestCreateCoreComponents(t *testing.T) {
+	t.Parallel()
+
+	t.Run("should work", func(t *testing.T) {
+		t.Parallel()
+
+		comp, err := CreateCoreComponents(createArgsCoreComponentsHolder())
```
Comment (iulianpascalau) [inline node/chainSimulator/components/coreComponents_test.go:113]: should we call comp.Close() at the end of this test?

File: node/chainSimulator/components/bootstrapComponents_test.go (hunk 1 above)
Comment (sstanculeanu) [reply node/chainSimulator/components/bootstrapComponents_test.go:135]: nope, added for coverage only

File: node/chainSimulator/components/coreComponents_test.go (hunk 2 above)
Comment (sstanculeanu) [reply node/chainSimulator/components/coreComponents_test.go:113]: nope, added for coverage only

File: node/chainSimulator/components/bootstrapComponents_test.go (hunk 1 above)
Comment (sstanculeanu) [reply node/chainSimulator/components/bootstrapComponents_test.go:135]: added everywhere

**Review by iulianpascalau — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Pull request merged**